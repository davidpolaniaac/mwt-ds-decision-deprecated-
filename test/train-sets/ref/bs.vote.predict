0.000000 0.000000 0.000000
0.041258 0.000000 0.165033
0.037014 0.000000 0.148058
0.014228 0.000000 0.056914
0.013842 0.000000 0.055368
0.027147 0.000000 0.108588
0.070507 0.060035 0.099882
0.196559 0.185384 0.214994
0.092980 0.076365 0.133533
0.180410 0.153715 0.229805
0.217660 0.151824 0.281808
0.195118 0.124296 0.267574
0.108614 0.052844 0.236524
0.225555 0.146848 0.297702
0.145296 0.095344 0.189743
0.308091 0.263104 0.362490
0.251460 0.188637 0.294482
0.336186 0.314464 0.363651
0.447857 0.391540 0.490716
0.193181 0.129572 0.261093
0.111861 0.069807 0.134286
0.293877 0.280419 0.559181
0.310941 0.203773 0.384707
0.241324 0.212224 0.261401
0.120889 0.110674 0.136587
0.383308 0.292162 0.489802
0.199919 0.138171 0.244588
0.281838 0.214734 0.363004
0.317083 0.244205 0.385861
0.316242 0.231640 0.377310
0.395086 0.346767 0.538611
0.196432 0.116643 0.256667
0.166444 0.135187 0.213753
0.305620 0.266950 0.357525
0.357438 0.326916 0.541221
0.283517 0.232655 0.516905
0.295835 0.223680 0.378183
0.358858 0.279042 0.426095
0.400030 0.358019 0.456708
0.533932 0.444014 0.563562
0.271197 0.208084 0.394655
0.314793 0.265202 0.434631
0.394829 0.330610 0.459317
0.252140 0.204864 0.334043
0.299280 0.264325 0.352090
0.207139 0.122604 0.278542
0.388121 0.337852 0.663552
0.411002 0.389132 0.541451
0.191517 0.065278 0.283866
0.165037 0.137934 0.193465
0.265873 0.223702 0.296520
0.301715 0.273999 0.323600
0.354242 0.339832 0.371714
0.348965 0.293943 0.427828
0.442426 0.427890 0.529465
0.612848 0.535167 0.703843
0.424167 0.393061 0.459777
0.293727 0.255470 0.342168
0.432392 0.382347 0.476401
0.332651 0.279962 0.380802
0.289300 0.266670 0.319516
0.399243 0.354260 0.441986
0.595021 0.400923 0.619577
0.225693 0.178755 0.332358
0.459629 0.434999 0.491165
0.260673 0.225289 0.315372
0.377175 0.300479 0.432123
0.689113 0.609933 0.803870
0.373654 0.305149 0.435862
0.386758 0.348280 0.558524
0.403237 0.382903 0.416826
0.423696 0.409965 0.560847
0.280904 0.181245 0.388152
0.300783 0.206158 0.387712
0.362861 0.206952 0.514328
0.341934 0.258637 0.408113
0.345541 0.175173 0.466962
0.558716 0.328643 0.587575
0.136375 0.125153 0.153853
0.188440 0.140399 0.238694
0.230628 0.210751 0.259890
0.294692 0.220530 0.347812
0.560731 0.537810 0.595470
0.315246 0.246706 0.371924
0.289648 0.167971 0.435135
0.275322 0.231589 0.307771
0.223535 0.214186 0.235897
0.603842 0.368226 0.654590
0.617442 0.539060 0.683148
0.313064 0.200537 0.395830
0.105966 0.072891 0.139037
0.324963 0.255444 0.503878
0.803859 0.718733 0.908420
0.197352 0.148134 0.234985
0.517387 0.419400 0.526940
0.372234 0.281590 0.449685
0.452235 0.416415 0.524149
0.223175 0.181038 0.257179
0.366673 0.288000 0.435358
0.325142 0.241666 0.383677
0.646592 0.542362 0.815245
0.554464 0.406909 0.555072
0.357697 0.307712 0.434762
0.431750 0.413690 0.456131
0.701642 0.394815 0.766386
0.480256 0.465606 0.558527
0.152626 0.123853 0.193798
0.391178 0.322638 0.425156
0.238568 0.152502 0.277716
0.556161 0.302211 0.603111
0.587753 0.519083 0.691395
0.584237 0.299794 0.647601
0.432170 0.384462 0.517525
0.271607 0.221309 0.305192
0.405185 0.367274 0.503765
0.426857 0.337746 0.591077
0.191398 0.145899 0.263625
0.575049 0.414074 0.599592
0.276198 0.152904 0.557253
0.378080 0.345800 0.410094
0.333159 0.283921 0.442343
0.452322 0.420802 0.540704
0.417493 0.382728 0.467407
0.433165 0.332877 0.487905
0.579312 0.483052 0.669946
0.357468 0.311442 0.509110
0.370780 0.304542 0.405826
0.657956 0.476059 0.722197
0.238837 0.087009 0.356681
0.325956 0.286310 0.554956
0.348132 0.262545 0.395812
0.773446 0.456271 0.832506
0.213320 0.162775 0.234764
0.286481 0.237567 0.308066
0.375162 0.304797 0.417123
0.248881 0.187232 0.322590
0.557487 0.509059 0.586988
0.415753 0.337405 0.492913
0.719985 0.637149 0.806485
0.338527 0.290388 0.402846
0.438398 0.409257 0.463217
0.260226 0.205731 0.290107
0.736141 0.703565 0.780184
0.834197 0.762319 0.883820
0.439510 0.430969 0.536015
0.582531 0.453805 0.627823
0.581004 0.421150 0.595646
0.831796 0.778521 0.896804
0.391706 0.263792 0.487996
0.395043 0.314232 0.442666
0.662890 0.340442 0.744408
0.327388 0.265696 0.386747
0.668315 0.588076 0.721545
0.481428 0.476270 0.573887
0.654967 0.537259 0.752284
0.602551 0.540044 0.642390
0.854136 0.789948 0.897156
0.555524 0.499521 0.624615
0.778131 0.674253 0.872025
0.590276 0.439344 0.610095
0.619377 0.407932 0.684588
0.331255 0.227526 0.417341
0.352056 0.245464 0.601820
0.680426 0.647146 0.707758
0.329759 0.292232 0.355189
0.420328 0.394226 0.478339
0.815523 0.729623 0.905942
0.358004 0.326497 0.618217
0.518260 0.424472 0.527758
0.752873 0.398577 0.780571
0.677538 0.644528 0.725458
0.334505 0.263530 0.378992
0.254666 0.215014 0.308513
0.354616 0.317181 0.408162
0.150307 0.124466 0.178991
0.576748 0.467242 0.638149
0.304070 0.281432 0.327047
0.878549 0.779583 0.985479
0.616192 0.539432 0.687287
0.745711 0.698828 0.777052
0.434255 0.396851 0.521240
0.385811 0.361413 0.418624
0.575764 0.443890 0.590839
0.351018 0.278819 0.443287
0.292144 0.163261 0.431669
0.333518 0.279712 0.564609
0.721920 0.600836 0.827837
0.611970 0.534030 0.688263
0.312084 0.198917 0.376790
0.665639 0.510130 0.786448
0.674845 0.450645 0.712192
0.275177 0.088898 0.485186
0.269720 0.209145 0.319710
0.288981 0.260771 0.315616
0.313221 0.256404 0.383797
0.465834 0.454846 0.706750
0.461448 0.446069 0.578452
0.350353 0.282540 0.459406
0.372410 0.254622 0.458421
0.586142 0.386176 0.671750
