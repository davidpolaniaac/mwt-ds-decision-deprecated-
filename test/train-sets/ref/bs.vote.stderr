final_regressor = models/bs.vote.model
Num weight bits = 18
learning rate = 0.5
initial_t = 0
power_t = 0.5
predictions = bs.vote.predict
using no cache
Reading datafile = train-sets/0001.dat
num sources = 1
average    since         example     example  current  current  current
loss       last          counter      weight    label  predict features
1.000000   1.000000            1         1.0   1.0000   0.0000       51
0.500851   0.001702            2         2.0   0.0000   0.0413      104
0.250819   0.000786            4         4.0   0.0000   0.0142      135
0.249190   0.247560            8         8.0   0.0000   0.1966      146
0.251954   0.254719           16        16.0   1.0000   0.3081       24
0.254812   0.257671           32        32.0   0.0000   0.1964       32
0.240693   0.226574           64        64.0   0.0000   0.2257       61
0.235389   0.230084          128       128.0   1.0000   0.6580      106

finished run
number of examples per pass = 200
passes used = 1
weighted example sum = 200
weighted label sum = 0
average loss = 0.209591
best constant = 0
total feature number = 15482
